chrome/content/test/add_reporters.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @fileOverview Defines the Jasmine Terminal Reporter.
+ * 
+ **/
+/* global jasmine, g */
+(function() {
+  var jasmineEnv = jasmine.getEnv();
+  jasmineEnv.updateInterval = 2500;
+  var terminalReporter = new jasmineReporters.TerminalReporter({
+                               verbosity: 3,
+                               color: "green",
+                             });
+  var old_callback = terminalReporter.jasmineDone;
+  terminalReporter.jasmineDone = function() {
+    old_callback();
+    // Close the browser on completing tests.
```

Comment:
nit: "above behaviour" is imprecise. Better wording would be "Comment the setTimeout if you don't want the window to close upon test completion."